token-repository/src/main/scala/io/renku/tokenrepository/repository/AccessTokenCrypto.scala

```diff
@@ -74,7 +74,7 @@ private class AccessTokenCryptoImpl[F[_]: MonadThrow](
     def maybeExtract(field: String, as: String => Either[IllegalArgumentException, AccessToken]) =
       cursor.downField(field).as[Option[String]].flatMap(to(as))
 
-    maybeExtract("projectAccessToken", as = ProjectAccessToken.from)
+    maybeExtract("project", as = ProjectAccessToken.from)
```

Comment:
Don't we have some old data in the db or somewhere that would still have `projectAccessToken`?